### PR TITLE
Collections

### LINKED ISSUE #16: Feature request: Collections module
Module for collection management is needed

### PR DESCRIPTION
**Pull request by skazska**
Added module Collections as cut copypaste of Devices module 
Closes #16

### CODE REVIEW

**Comment by skazska** [comment]
Hi everyone, is there any issues with this pull request? when could it be took in consideration to merge it?

**Comment by matflores** [comment]
@skazska thanks for your contribution! Looks good to me :+1: 

@stream7 could you please review and merge when you have a chance? Thanks!

File: lib/collections.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Collections.prototype.metadata = function(id, callback) {
+  return this.client.get(helpers.url("/collections/%s/metadata", id), callback);
+};
+
+// Update the metadata of the collection
+//
+// https://m2x.att.com/developer/documentation/v2/collections#Update-Collection-Metadata
+Collections.prototype.updateMetadata = function(id, params, callback) {
+  return this.client.put(
+    helpers.url("/devices/%s/metadata", id),
```
Comment (stream7) [inline lib/collections.js]: This should be `helpers.url("/collections/%s/metadata", id),`

File: lib/collections.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+/**
+ * Created by ska on 9/14/15.
+ */
+"use strict";
+
+var helpers = require("./helpers");
+var Collections;
+
+// Wrapper for AT&T M2X Device API
```
Comment (stream7) [inline lib/collections.js]: Could you also update these references to Device docs? There a few more.

**Comment by skazska** [comment]
@stream7  thanks for your remarks,  i fixed them

**Comment by stream7** [comment]
@skazska excellent! Thank you!

**Pull request merged**